Library compile issue (Android)

error: lambda expressions are not supported in -source 1.7
        mainHandler.post(() -> promise.resolve(fibonacci(intensity)));
                            ^
  (use -source 8 or higher to enable lambda expressions)
1 error

FAILURE: Build failed with an exception.

I am getting this when trying to build the app on my android device. Any solutions?

--- a/flipper-native/android/build.gradle
+++ b/flipper-native/android/build.gradle
@@ -14,6 +14,10 @@ android {
         versionCode 1
         versionName "1.0"
     }
+    compileOptions {
+        sourceCompatibility JavaVersion.VERSION_1_8
+        targetCompatibility JavaVersion.VERSION_1_8
+    }
 }
 
 dependencies {

